awesomemotive/affiliatewp-allow-own-referrals
# Issue/21 - Bump WordPress minimum requirements

Issue #21


**Housekeeping**


**Proposed changes**


## How to test it
1. Install and activate Allow Own Referrals add-on
2. Confirm the plugin works as expected without any PHP errors.
